newsletter-february-2023.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: Feb 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - February 2023
+========================================
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-january-2023>`:
+
+- 🪄️ Build outputs are now stored in a well-known location, available to the build process.
+  This opens up many new and exciting possibilities for generating and processing final output formats,
+  which we will uncover in an upcoming blog post.
+  PDFs for MkDocs and encrypted documentations are just two demos that we have ready.
+  **Stay tuned!**
+- 🌪️ Uploads are faster and done in parallel,
+  thanks to `rclone`_. 
+- 🔒️ CORS headers have been tightened on all platforms.
+- 🛠️ The default branch of a git repository is now correctly detected in manual imports.
```

Comment:
```suggestion
- 🛠️ The default branch of a Git repository is now correctly detected in manual imports.
```

:)

Reply:
he he, multiple Anthony suggestions on this over time made us to improve our eyes and detect this immediately now 😄